sound/soc/sof/intel/hda-dai.c

```diff
@@ -0,0 +1,132 @@
+// SPDX-License-Identifier: (GPL-2.0 OR BSD-3-Clause)
+/*
+ * This file is provided under a dual BSD/GPLv2 license.  When using or
+ * redistributing this file, you may do so under either license.
+ *
+ * Copyright(c) 2018 Intel Corporation. All rights reserved.
+ *
+ * Authors: Keyon Jie <yang.jie@linux.intel.com>
+ */
+
+#include <sound/pcm_params.h>
+#include "../sof-priv.h"
+
+/* common dai driver for skl+ platforms */
+static const struct snd_soc_dai_ops skl_dai_ops = {
+};
+
+#define SKL_FORMATS (SNDRV_PCM_FMTBIT_S16_LE | SNDRV_PCM_FMTBIT_S24_LE | \
+	SNDRV_PCM_FMTBIT_S32_LE | SNDRV_PCM_FMTBIT_FLOAT)
+
+/* Skylake+ DAIs */
```

Comment:
It's probably worth commenting here that some products who use this DAI array only physically have a subset of the DAIs, but no harm is done here by adding the whole set.

Reply:
done.